package.json

```diff
@@ -45,9 +45,11 @@
     "CMakeLists.txt"
   ],
   "scripts": {
+    "lerna:bootstrap": "lerna bootstrap",
```

Comment:
It's more convenient to type `npx lerna bootstrap` than `npm run lerna:bootstrap`.  How about `npm run bootstrap`?

Reply:
You can still type `npx lerna bootstrap`? I think `bootstrap` as a package script name is a bit too generic.

Reply:
I agree.  Then I will just keep using `npx lerna bootstrap`